diff --git a/src/modules/espeak.c b/src/modules/espeak.c
index be96cd2..22cbd2b 100644
--- a/src/modules/espeak.c
+++ b/src/modules/espeak.c
@@ -612,7 +612,7 @@ espeak_set_rate(signed int rate)
 {
 	assert(rate >= -100 && rate <= +100);
 	int speed;
-	int normal_rate = 170, max_rate = 390, min_rate = 80;
+	int normal_rate = 175, max_rate = 450, min_rate = 80;
     
 	if (rate < 0)
 		speed = normal_rate + (normal_rate - min_rate) * rate / 100;
